discourse_rock/patches/lp1903695.patch

```diff
@@ -1,11 +1,13 @@
+diff --git a/lib/middleware/anonymous_cache.rb b/lib/middleware/anonymous_cache.rb
+index d41069c92e..fe968c6d64 100644
 --- a/lib/middleware/anonymous_cache.rb
 +++ b/lib/middleware/anonymous_cache.rb
-@@ -318,7 +318,7 @@ module Middleware
+@@ -347,7 +347,7 @@ module Middleware
+       return @app.call(env) if defined?(@@disabled) && @@disabled
  
-     def call(env)
        if PAYLOAD_INVALID_REQUEST_METHODS.include?(env[Rack::REQUEST_METHOD]) &&
--        env[Rack::RACK_INPUT].size > 0
-+        env[Rack::RACK_INPUT].respond_to?(:size) && env[Rack::RACK_INPUT].size > 0
```

Comment:
We probably don't need this anymore

Reply:
Might be, I would like to tackle that in another story though. Looking at the bug on Launchpad I don't understand why/how the bug creeps in, whether a bug/issue upstream is necessary, so I really want to dig into this once there is a running rock with Discourse v3.

Reply:
@arturo-seijas the Discourse code still lacks that piece of code, so I have opted for keeping it FYI.